packages/react-core/src/components/DataList/examples/DataList.md

```diff
@@ -27,1237 +27,49 @@ import { css } from '@patternfly/react-styles';
-      ? `Move ${items[source.index].content} to ${items[dest.index].content}`
-      : 'Invalid drop zone';
-    if (newText !== liveText) {
-      setLiveText(newText);
-    }
-  }
-
-  function onDrop(source, dest) {
-    if (dest) {
-      const newItems = reorder(
-        items,
-        source.index,
-        dest.index
-      );
-      setItems(newItems);
-
-      setLiveText('Dragging finished.');
-      return true; // Signal that this is a valid drop and not to animate the item returning home.
-    } else {
-      setLiveText('Dragging cancelled. List unchanged.');
-    }
-  }
-
-  return (
-    <DragDrop onDrag={onDrag} onDragMove={onDragMove} onDrop={onDrop}>
-      <Droppable hasNoWrapper>
-        <DataList aria-label="draggable data list example" isCompact>
-          {items.map(({id, content}) =>
-            <Draggable key={id} hasNoWrapper>
-              <DataListItem aria-labelledby={id} ref={React.createRef()}>
-                <DataListItemRow>
-                  <DataListControl>
-                    <DataListDragButton
-                      aria-label="Reorder"
-                      aria-labelledby={id}
-                      aria-describedby="Press space or enter to begin dragging, and use the arrow keys to navigate up or down. Press enter to confirm the drag, or any other key to cancel the drag operation."
-                      aria-pressed="false"
-                    />
-                    <DataListCheck aria-labelledby={id} name={id} otherControls />
-                  </DataListControl>
-                  <DataListItemCells
-                    dataListCells={[
-                      <DataListCell key={id}>
-                        <span id={id}>{content}</span>
-                      </DataListCell>
-                    ]}
-                  />
-                </DataListItemRow>
-              </DataListItem>
-            </Draggable>
-          )}
-        </DataList>
-      </Droppable>
-      <div className="pf-screen-reader" aria-live="assertive">
-        {liveText}
-      </div>
-    </DragDrop>
-  );
-};
-```
-
-### Small grid breakpoint
-
-```js
-import React from 'react';
-import {
-  DataList,
-  DataListItem,
-  DataListItemRow,
-  DataListItemCells,
-  DataListCell
-} from '@patternfly/react-core';
 
-<DataList aria-label="Simple data list example" gridBreakpoint="sm">
-  <DataListItem aria-labelledby="simple-item1">
-    <DataListItemRow>
-      <DataListItemCells
-        dataListCells={[
-          <DataListCell key="primary content">
-            <span id="simple-item1">Primary content</span>
-          </DataListCell>,
-          <DataListCell key="secondary content">
-            Really really really really really really really really really really really really really really long
-            description that should be truncated before it ends
-          </DataListCell>
-        ]}
-      />
-    </DataListItemRow>
-  </DataListItem>
-</DataList>
+```ts isBeta file="./DataListDraggable.tsx"
```

Comment:
Looks like you are missing the `DataListSmGridBreakpoint.tsx` file in the MD file.

Reply:
Nice catch @tlabaj - thank you! Updated.